lib/Gym/space_handler.ex

```diff
@@ -0,0 +1,28 @@
+defmodule Gyx.Gym.Utils do
+  @moduledoc """
+  This module contains auxiliary functions to achieve full
+  compatibility with Gym, including functions to obtain
+  Gyx space representations from Gym space specs.
+  """
+  @space_types [{~r/Discrete\((?<n>\d+)\)/, :discrete}]
+
+  @doc """
+  This function takes the __repr__ response from Gym spaces
+  and creates an equivalent Gyx Space struct
+  """
+  def gyx_space(gym_space_string) do
+    gym_space_string |> parse |> create_space
+  end
+
+  def parse(gym_space_string) do
+    {regex, type} =
+      Enum.find(@space_types, {nil, :unknown}, fn {regex, _} ->
```

Comment:
no entiendo que hace esta función... tal vez un mejor nombre ayudaría.. pero de preferencia, documentar todas las funciones públicas..

Reply:
It would iterate all ` {<regex>, <space_type>} ` on `@space_types` to match the corresponding regex